GooglePayment/build.gradle

```diff
@@ -7,12 +7,13 @@ android {
     compileSdkVersion 28
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 21
```

Comment:
@Epreuve @demerino Do you foresee any issues with bumping up the minSDKVersion? I'm not sure how this was working previously, since the base SDK has a minSDKVersion of 21, and I believe this module depends on it.

Reply:
Is it possible to use this module outside of the BT SDK? If so, it's fine to bump the min sdk version, but you'll also need to do a major version bump of this module.

Reply:
We just chatted with Quinn, and his suggestion was to increase the `minSdkVersion` to 21 and do a major version bump of this module.

Technically, you can include this module in your project without including `braintree-android`, but you wouldn't be able to use it for anything, since all of the methods require a `BraintreeFragment`.